### ISSUE TITLE
Net sales typo in dashboard. It should be Gross sales not "Net".

### ISSUE THREAD

**Issue by relogical** [woocommerce/woocommerce#13012]
## SYSTEM STATUS REPORT

```
Grab the system status report from WooCommerce > System Status and paste it here.
```

**Comment by relogical** [comment]
Net is incorrect. Net sales are calculated after profit. Since Woocommerce does not calculate cost of product it needs to show that number as Gross.

**Comment by mikejolley** [comment]
Nope, the net value matches the net value on the main reports page. We show net here, not gross.

**Comment by relogical** [comment]
I understand but you are referring to this value as Net. It should be Gross.

**Comment by relogical** [comment]
I understand what you are saying. But in accounting Net sales are calculated after the cost of product has been subtracted. Since cost of product is not calculated it is incorrect to call this Net.

**Comment by mikejolley** [comment]
This is not accounting software. We don't track how much products are worth. We still need to distinguish the two values. Net sales makes sense to me...
